app/processing/payment/get-latest-payment.js

```diff
@@ -2,16 +2,16 @@ const moment = require('moment')
 const getPaymentSchedule = require('./get-payment-schedule')
 const validateLatestPayment = require('./validate-latest-payment')
 
-const getLatestPayment = (paymentRequest, settlement, lastSettlement) => {
-  const lastSettlementValue = lastSettlement?.value ?? 0
-  const paymentValue = settlement.value - lastSettlementValue
+const getLatestPayment = (paymentRequest, settlement, supportingSettlements) => {
+  const supportingPaymentValue = supportingSettlements?.reduce((total, settlement) => total + settlement.paymentValue, 0) ?? 0
+  const adjustedPaymentValue = settlement.paymentValue + supportingPaymentValue
```

Comment:
Need to include PPA settlement chains in order to get overall payment value.

As last settlement is needed twice earlier in the process, have moved it to the settlements module as a reusable function.